src/index.scss

```diff
@@ -92,29 +92,22 @@ body {
       height: 100%;
       mix-blend-mode: multiply;
     }
-
-    .drawing-cancelation-button {
-      pointer-events: auto;
-      position: sticky;
-      bottom: 1rem;
-      left: 1rem;
-      font-size: 1rem;
-      max-width: 10rem;
-      text-align: left;
-      outline: 0;
-      padding: 0.5rem;
-      margin: 0;
-      border: 1px solid black;
-      border-radius: 0.25rem;
-      background: #fff;
-
-      &:hover {
-        background: #eee;
-      }
-    }
   }
 }
 
+.drawing-completion-popover {
+  pointer-events: auto;
+  position: sticky;
+  bottom: 0;
```

Comment:
one last thing: I want it to float a bit more instead of being aligned to the bottom of the viewport. can you try a `0.5rem` or `1rem` here so it's more...floaty?

Reply:
yup, not sure why I set those bottom/left attributes to zero

originally I had a big change trying to get it to display more nicely, but gave up as the state management was tied into the hierarchy too much for me to change